storage/blockblob.go

```diff
@@ -77,10 +79,17 @@ func (b *Blob) CreateBlockBlob(options *PutBlobOptions) error {
 // reader. Size must be the number of bytes read from reader. To
 // create an empty blob, use size==0 and reader==nil.
 //
+// Any headers set in blob.Properties or metadata in blob.Metadata
+// will be set on the blob.
+//
 // The API rejects requests with size > 256 MiB (but this limit is not
 // checked by the SDK). To write a larger blob, use CreateBlockBlob,
 // PutBlock, and PutBlockList.
 //
+// To create a blob from scratch, call container.GetBlobReference() to
```

Comment:
This is correct

---
In reply to: [130678050](https://github.com/Azure/azure-sdk-for-go/pull/712#discussion_r130678050) [](ancestors = 130678050)